ci/serverless/kibana_api_tests.sh

```diff
@@ -7,7 +7,7 @@ export PIPELINE_NAME="stdin_stdout"
 export EXIT_CODE="0"
 
 create_pipeline() {
-    RESP_CODE=$(curl -s -w "%{http_code}" -o /dev/null -X PUT -H "Authorization: ApiKey $LS_ROLE_API_KEY_ENCODED" "$KB_ENDPOINT/api/logstash/pipeline/$PIPELINE_NAME" \
+    RESP_CODE=$(curl -s -w "%{http_code}" -o /dev/null -X PUT -H "Authorization: ApiKey $TESTER_API_KEY_ENCODED" "$KB_ENDPOINT/api/logstash/pipeline/$PIPELINE_NAME" \
```

Comment:
Should these test cases have their own API key, that only has `manage_logstash_pipelines` privs, rather than using the broader`tester` user?

Reply:
At the moment, these APIs are not ready for test in QA and Logstash is not the direct user, therefore I cannot find the minimum privileges. Kibana test is disabled.

Reply:
added a comment in the file